src/components/SharedComponents/Button/common.ts

```diff
@@ -197,6 +200,23 @@ const BUTTON_CONFIGS: Record<
   "xs-buttonStyledAsLink-icon-button": DEFAULT_BUTTON_CONFIG,
   "xs-flat-button": DEFAULT_BUTTON_CONFIG,
   "xs-flat-icon-button": DEFAULT_BUTTON_CONFIG,
+  "medium-brush-button": DEFAULT_BUTTON_CONFIG,
```

Comment:
I haven't used the old style components enough, can you explain why all these are required?

Reply:
I was receiving TypeScript errors unless I defined all permutations of this new medium variants. This is necessary because the button have multiple variants (brush, minimal, flat etc.) and sizes (xs to large) – which all require a default configurations. I required a medium configuration in the first place because the icon size in Figma for the CTA fell in between the `small` and `large` size.